build.properties.dist

```diff
@@ -110,6 +110,12 @@ behat.yml.path = ${behat.dir}/behat.yml
 # The base URL to use in Behat tests.
 behat.base_url = http://localhost
 
+# The URL of the Behat webdriver host.
+behat.wd_host.url = http://localhost:8643/wd/hub
```

Comment:
Why not use 4444 ? Everyone uses that on the internet, why such a change ? I understand it's not a big deal, I'm just trying to understand :)